pipeline/alert.go

```diff
@@ -942,6 +947,23 @@ func (a *AlertaHandler) Services(service ...string) *AlertaHandler {
 	return a
 }
 
+// Send alert to an MQTT broker
+// tick:property
+func (a *AlertNode) MQTT() *MQTTHandler {
```

Comment:
Can we accept the topic string here in this method? That way it reads like 

```
|alert()
    .mqtt('topic goes here')
```

Since there is no other config for the MQTT handler this keeps it simple.